tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/AbstractPitclipseSWTBotTest.java

```diff
@@ -356,6 +355,41 @@ protected static void consoleContains(int generatedMutants, int killedMutants,
         }
     }
 
+    private static String getConsoleText() {
+        SWTBotView consoleView = bot.viewByPartName("Console");
+        consoleView.show();
+        return consoleView.bot()
+                .styledText().getText()
+                .replace("\r", "");
+    }
+
+    /**
+     * Asserts that the only active mutator was the given mutator.
+     * @param mutators which should be the only active mutator
+     */
+    protected static void mutatorIs(Mutators mutators) {
+        final String consoleText = getConsoleText();
```

Comment:
Remember that you must make sure that PIT has been completed on the Console, that's why in the method above we have `PAGES.views().waitForTestsAreRunOnConsole();`. Maybe you may want to put that wait in the method you extracted `getConsoleText`?

Reply:
In case, please also add a Javadoc to `getConsoleText`